src/Models/PhoneLink.php

```diff
@@ -19,6 +19,11 @@ class PhoneLink extends Link
      * Set the priority of this link type in the CMS menu
      */
     private static int $menu_priority = 40;
+    
+    /**
+     * Set the default icon for the Phone link type
+     */
+    private static $icon = 'font-icon-block-phone';
```

Comment:
```suggestion
    private static $icon = 'font-icon-mobile';
```
This is fairly subjective - but I think the mobile icon is easier to see (the box around the block-phone icon makes the actual phone icon really small).
![the "block-phone" and "mobile" icons side-by-side](https://github.com/silverstripe/silverstripe-linkfield/assets/36352093/52f82955-d5d0-499b-8204-385875916336)

Reply:
Updated